anotacoes/api.md

```diff
@@ -0,0 +1,125 @@
+api.md
```

Comment:
Dúvida: Talvez tenha deixado passar esta frase. 

Sugestão: No início realmente é interessante programar sem auxiliares, em alguns cenários ele acaba atrapalhando, porém utilizo uma extensão no VScode que me ajuda com erros de miss click.

https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Reply:
Obrigada!!